runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -217,11 +220,31 @@ public void onGameTick(final GameTick event)
 
 			if (objectId != -1)
 			{
-				GameObjectQuery query = new GameObjectQuery().idEquals(objectId);
-				objectsToMark = queryRunner.runQuery(query);
+				if (clue instanceof LocationClueScroll)
+				{
+					// Match object with location every time
+					final WorldPoint location = ((LocationClueScroll) clue).getLocation();
+					LocalPoint localLocation = LocalPoint.fromWorld(client, location);
```

Comment:
Doesn't this result in an NPE on clues that are missing a location (getLocation() returns null)?

Reply:
Yes it does, oops.